packages/odyssey-react/src/components/Form/Form.module.scss

```diff
@@ -11,12 +11,12 @@
  */
 
 .root {
-  max-width: calc(#{$max-line-length} + (2 * #{$spacing-m}));
-  margin-block-start: 0;
-  margin-block-end: $spacing-m;
-  margin-inline: 0;
-  padding-block: $spacing-m;
-  padding-inline: $spacing-m;
+  max-width: calc(var(--FontLineLengthMax) + (2 * var(--SpacePaddingInline)));
```

Comment:
thinking ahead here not related to your PR changes directly: we need to make time to audit our use of `calc`. We've talked about this in PR feedback already but the more micro adjustments we make the further away from the consistant application of token values we get. Having them derived from other token values at run time is OK, but in many cases I would think we could extract these out into static values as well which helps run time performance and keeps the maintenance overhead lower as the values are more human friendly and don't require math. cc @okta/design-system